src/locale/misc/smartSearch/en.yml

```diff
@@ -11,6 +11,7 @@ filterTitles:
     all: All people
     campaign_participation: People who have participated in a campaign
     most_active: A selection of the most active people
+    person_data: People who match some criteria
```

Comment:
This label is true for all the filters. Maybe replace it with something like "Search based on name, e-mail or similar fields"?